src/ZeDMD.cpp

```diff
@@ -483,7 +486,11 @@ void ZeDMD::RenderRgb565(uint16_t* pFrame)
 
   int rgb565Size = Scale16(m_pPlanes, pFrame, &width, &height, is_bigendian());
 
-  if (m_usb)
+  if (m_wifi)
+  {
+    m_pZeDMDWiFi->QueueCommand(ZEDMD_COMM_COMMAND::RGB565ZonesStream, m_pPlanes, rgb565Size, width, height, 2);
+  }
+  else if (m_usb)
   {
     m_pZeDMDComm->QueueCommand(ZEDMD_COMM_COMMAND::RGB565ZonesStream, m_pPlanes, rgb565Size, width, height, 2);
   }
```

Comment:
@bartdesign  - isn't this the same line? ie, if (m_wifi || m_usb) ?

Reply:
Maybe i don't understand the question, but this is the same way it has been done before in the RGB24Streaming method.

Reply:
your right. I looked too fast, and didn't see `m_pZeDMDWiFi` + `m_pZeDMDComm`. All good.